test/e2e/instance-serial.e2e.ts

```diff
@@ -19,7 +19,7 @@ test('serial console can connect while starting', async ({ page }) => {
   // now go starting to its serial console page while it's starting up
   await expect(page).toHaveURL('/projects/mock-project/instances/abc/storage')
   await page.getByRole('tab', { name: 'Connect' }).click()
-  await page.getByRole('link', { name: 'Connect' }).click()
+  await page.getByRole('link', { name: 'Connect' }).nth(1).click()
```

Comment:
This is fine, but another approach that might work is use `main` or the tab content pane to select a container and then the connect link.